src/Reader/RowIteratorInterface.php

```diff
@@ -8,9 +8,9 @@
 use OpenSpout\Common\Entity\Row;
 
 /**
- * @extends Iterator<Row>
+ * @extends Iterator<null|Row>
```

Comment:
is the Row actually nullable during iteration ? Or can the `current()` function return null only for iterators that are not in the valid state ? If the iteration never returns null, this line should be reverted.

From looking at the code, the rowBuffer cannot be null when the `valid()` method returns `true`.

Reply:
You are right, only a raw call to `current()` method can return `null`, the Iterator itself never iterates over `null` values.

Can you pick this fix up please?